src/main/java/Model/User.java

```diff
@@ -0,0 +1,42 @@
+package Model;
+
+public class User {
```

Comment:
Why this new class?

Reply:
Need to create a separate package and individual classes for Models